docs/docs/writing-hello-world.md

```diff
@@ -262,20 +222,7 @@ java -jar host/build/libs/host-mock.jar
 
 ## Write the client
 
-The client app will do three things:
-
-1. Connect to the host web server and download the [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) from it.
-1. Verify the enclave is acceptable: i.e. that it will do what's expected.
-1. Send it the command line arguments as a string to reverse and get back the answer, using encrypted mail.
-
-Actually, the first two steps are done for you when using an
-[`EnclaveClient`](api/-conclave%20-core/com.r3.conclave.client/-enclave-client/index.html) object. `EnclaveClient` handles the encryption and
-decryption of Mail for you and provides a simple interface for sending and receiving Mail. However, it doesn't know
-_how_ to transport the mail, which is where [`EnclaveTransport`](api/-conclave%20-core/com.r3.conclave.client/-enclave-transport/index.html)
-comes in. Since we're connecting to the host web server, we'll be using
-[`WebEnclaveTransport`](api/-conclave%20-core/com.r3.conclave.client.web/-web-enclave-transport/index.html) as our transport.
-
-Replace the client code at `client/src/.../ReverseEnclaveClient.java`
+Replace the client code at `client/src/.../ReverseEnclaveClient.java` with:
```

Comment:
Maybe it is better to provide the complete path? Not sure if there is a convention

Reply:
We use a similar path in another place in the doc too. I think it's okay. It also allows flexibility if any subfolder changes in the future.